src/command/command.ts

```diff
@@ -73,19 +63,18 @@ export class Command<TContext extends IContext = IContext>
 
   public static createMatcherFromString(
     pattern: string,
-    params: ICreateMatcherFromStringParams = {},
+    params: CreateMatcherFromStringParams = {},
   ): CommandMatcher {
     if (typeof pattern === undefined) {
       return () => 0;
     }
 
     pattern = pattern ? pattern.toLowerCase() : '';
-    const {
-      relevanceProvider = getLevenshteinRelevance,
-    } = params;
-    return (context: IContext) => {
-      const commandLower = context.data.request.command ?
-          context.data.request.command.toLowerCase() : '';
+    const { relevanceProvider = getLevenshteinRelevance } = params;
+    return (context: Context) => {
+      const commandLower = context.data.request.command
```

Comment:
This code duplicates at least 3 times, maybe it's better to implement function for this operation